transformers/events/log_item_update/transformer.go

```diff
@@ -0,0 +1,64 @@
+package log_item_update
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+	"github.com/makerdao/vulcanizedb/pkg/eth"
+)
+
+const OfferId event.ColumnName = "offer_id"
```

Comment:
Maybe this belongs in `transformers/shared/constants/keys.go`?

Reply:
That makes sense. I left it in the this transformer for now, since there are no other transformers with a field called `offer_id` but that's likely to change as we keep working through these Oasis events.